pkg/http/api.go

```diff
@@ -274,6 +276,17 @@ func (a *api) constructHealthzEndpoints() []Endpoint {
 	}
 }
 
+func (a *api) constructTransactionEndpoints() []Endpoint {
+	return []Endpoint{
+		{
```

Comment:
This api is part of state endpoint. so it needs to be declared in `constructStateEndpoints` function instead of defining the separate function.